conn2res/tasks.py

```diff
@@ -179,7 +179,7 @@ def name(self, name):
 
         self._name = name
 
-    def fetch_data(self, n_trials=None, horizon=1, **kwargs):
+    def fetch_data(self, n_trials=None, horizon=1, win=30, **kwargs):
```

Comment:
Before, when I wasnted to change from memory to prediction task I just flipped `x` and `y`. Now, I make sure that `x` is the same in both memory and prediction tasks so that the results can be compared more directly. To be able to do so, I introducted `win` which defines an initial window to be dropped from `x`: this way `y` can take those values if we use a prediction task. Consequently, `win` should be always equal or larger than `horizon` and the code throws an error if that is not the case.